Program.cs

```diff
@@ -15,252 +15,273 @@ public static void printsContentsRefrigerator(List<Refrigerator> allfrige)
+            if (!(int.TryParse(userInput, out kashrot)) || kashrot < 1 || kashrot > 3)
+                return -1;
+            return --kashrot;
+
+        }
+        public static int GetTypeItemInput()
         {
             int typeItem;
-            while (true)
-            {
-                Console.WriteLine("lease select the Type of the item:");
-                Console.WriteLine("1. " + (Item.TypeItem)0);
-                Console.WriteLine("2. " + (Item.TypeItem)1);
-                string inp = Console.ReadLine();
-                if (int.TryParse(inp, out typeItem) && typeItem > 0 && typeItem < 3)
-                {
-                    return (Item.TypeItem)(--typeItem);
-                }
-                else { Console.WriteLine("Please,enter valid input"); }
-            }
+            string userInput;
+            Console.WriteLine("Pease select the Type of the item:");
+            Console.WriteLine("1. " + (Item.TypeItem)0);
+            Console.WriteLine("2. " + (Item.TypeItem)1);
+            userInput = Console.ReadLine();
+            if (!(int.TryParse(userInput, out typeItem)) || typeItem < 1 || typeItem > 2)
+                return -1;
+            return --typeItem;
         }
-        public static String checkNameItemInput()
+        public static String GetNameItemInput()
         {
             string nameInput = "";
             Console.WriteLine("Name of the Item");
             nameInput = Console.ReadLine();
             while (nameInput.Length <= 0 || nameInput.Length > 50)
-                {
-                    Console.WriteLine("Please,enter valid input");
-                    nameInput = Console.ReadLine();
-                }
+            {
+                Console.WriteLine("Please,enter valid input");
+                nameInput = Console.ReadLine();
+            }
             return nameInput;
 
-            
+
         }
-        public static DateTime checkExpiryDatedInput()
+        public static object GetExpiryDatedInput()
         {
             string expiryDateInput;
             DateTime expiryDate = DateTime.Now;
             Console.WriteLine("Please enter the expiry date of the item in this format (MM/dd/yyyy):");
             Console.WriteLine("Dont enter a date before the year 2000");
             expiryDateInput = Console.ReadLine();
-            while ((!(DateTime.TryParseExact(expiryDateInput, "MM/dd/yyyy", CultureInfo.InvariantCulture, DateTimeStyles.None, out expiryDate))) || expiryDate.Year < 2000)
+            if ((!(DateTime.TryParseExact(expiryDateInput, "MM/dd/yyyy", CultureInfo.InvariantCulture, DateTimeStyles.None, out expiryDate))) || expiryDate.Year < 2000)
             {
-                Console.WriteLine("Please,enter valid input ");
-                expiryDateInput = Console.ReadLine();
+                return -1;
 
             }
             return expiryDate;
         }
-        public static int checkSizeItemInput()
+        public static int GetSizeItemInput()
```

Comment:
avoid static method in such an exercise like this.
the main purpose of static is to exposure the method to all layers of the code.